x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -14,6 +17,11 @@ func (k Keeper) ValidateInbound(
 	msg *types.MsgVoteInbound,
 	shouldPayGas bool,
 ) (*types.CrossChainTx, error) {
+	err := k.CheckMigration(ctx, msg)
+	if err != nil {
+		return nil, err
```

Comment:
Can we add a test for handling this error?

Reply:
ba3540db75230d2a96bfd959b31b7df7eb8b99df